frontend/packages/ceph-storage-plugin/src/components/kms-config/vault-auth-methods.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import { Trans } from 'react-i18next';
+
+import {
+  InputGroup,
+  FormGroup,
+  TextInput,
+  Button,
+  ValidatedOptions,
+  Tooltip,
+} from '@patternfly/react-core';
+import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+import { ExternalLink } from '@console/internal/components/utils';
+
+import { VaultConfig } from '../../types';
+
+export const VaultTokenConfigure: React.FC<VaultAuthMethodProps> = ({
+  t,
+  className,
+  vaultState,
+  setAuthValue,
+  isValid,
+}) => {
+  const [revealToken, setRevealToken] = React.useState(false);
+
+  return (
+    <FormGroup
+      fieldId="vault-token"
+      label={t('ceph-storage-plugin~Token')}
+      className={`${className}__form-body`}
+      helperTextInvalid={t('ceph-storage-plugin~This is a required field')}
+      validated={isValid(vaultState.authValue?.valid)}
+      helperText={
+        <Trans ns="ceph-storage-plugin">
+          Create a secret with the token for every namespace using encrypted PVCs. <br />
+          For more information, please refer to the <ExternalLink href="">document</ExternalLink>
```

Comment:
Lets not add any doc link for now. We need to get persistent doc links and check with doc team, this is in process.

Reply:
ack